lib/column/fixed_string.go

```diff
@@ -150,6 +203,8 @@ func (col *FixedString) Append(v any) (nulls []uint8, err error) {
 func (col *FixedString) AppendRow(v any) (err error) {
 	data := make([]byte, col.col.Size)
 	switch v := v.(type) {
+	case []byte:
+		copy(data, v)
```

Comment:
Why copy instead of assign slice?

Reply:
The length of slice may not equal col.col.Size.  Use copy to cover overlap scenarios